packages/manager/apps/web/client/app/domain/list/list-domain-layout.controller.js

```diff
@@ -94,6 +95,31 @@ export default class ListDomainLayoutCtrl extends ListLayoutHelper.ListLayoutCtr
     };
   }
 
+  getDate(domain, key) {
+    const date = new Date(domain[key]);
+    const locale = this.coreConfig.getUserLocale().replace('_', '-');
+    if (
+      key === 'renewalDate' &&
+      domain.renewalState === this.DOMAIN_RENEWAL_MODE.AUTOMATIC_RENEW
+    ) {
+      const formattedDate = new Intl.DateTimeFormat(locale, {
+        month: '2-digit',
+        year: 'numeric',
+      }).format(date);
```

Comment:
Wouldn't you get "today" date if domain.renewalDate is nil / not set ?
If so is it acceptable ?

Reply:
There is a test at function's final return, if renewalDate is null, returned date is an empty string :
` return domain[key]
      ? new Intl.DateTimeFormat(locale, {
          day: '2-digit',
          month: '2-digit',
          year: 'numeric',
        }).format(date)
      : '';`

Reply:
Nope if key === 'renewalDate' &&
      domain.renewalState === this.DOMAIN_RENEWAL_MODE.AUTOMATIC_RENEW
you use another return